### DISCUSSION TITLE
Guest/Anonymous user

### DISCUSSION THREAD

**Discussion by tomgroenwoldt** [goauthentik/authentik#6991] [Q&A]
Hey! :)

Is authentik capable of defining something like a guest user? I use it as a middleware between traefik and my web application and so far it worked out great. My web application uses the authentik authentication headers. I imagined some additional field like "Continue as guest" for my login page. The client would then login as the guest user and I can restrict access to that specific guest user by parsing the authentik header fields.
I didn't find anything about that topic so I am asking here.

Thanks in advance!

**Comment by semasimo** [comment]
I am thinking about something simmelar right now.

but i want the guest user to be transparent and access the page like normal, like it would be without using the proxy provider at all.

but if a user is already authed in my authentik i would want the headers to be send to my application so i can enchrich some more data, which i dont want everyone to see
